content/using/operations/using-bridge.md

```diff
@@ -73,3 +73,39 @@ From the detail page associated with your point, click the `Generate Arvo Keyfil
 Click `Generate ->`, which will download a keyfile onto your machine.
 
 With that keyfile in hand, you can now exit Bridge and continue to the guide to [install the Urbit binary](@/using/install.md).
+
+### Escaping your sponsor
+
+To use the network as a planet or star, you must be sponsored by an active star
```

Comment:
"To use the network" feels a tad broad, but not sure if specificity is useful/desired in this context.